pkgs/development/tools/misc/gnum4/doc.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, gnum4, texinfo4 }:
+stdenv.mkDerivation {
+  inherit (gnum4) src version;
+  pname = "gnum4-doc";
+  outputs = [ "out" "html" "txt" ];
+  phases = [ "unpackPhase" "buildPhase" "installPhase" ];
```

Comment:
Override `phases` is discouraged as it implicitly disables some less-known but useful phases. Attributes like `dontConfigure` are to be used instead.